sys/net/gnrc/sock/gnrc_sock.c

```diff
@@ -117,6 +117,12 @@ ssize_t gnrc_sock_recv(gnrc_sock_reg_t *reg, gnrc_pktsnip_t **pkt_out,
 #ifdef MODULE_XTIMER
     xtimer_t timeout_timer;
 
+    /* xtimer_spin would make this never receive anything.
```

Comment:
```suggestion
    /* xtimer_spin would make this never receive anything. */
```

Reply:
somehow the / made it back in again when adding the linebreak in front of the Avoid